### PR TITLE
Added screenshottaker program

### PR DESCRIPTION
**Pull request by iamvs-2002**
Fixes #
#32 
### Checklist:
- [x] My branch is up-to-date with upstream/master branch.

### Description:

Added a java program to take the screenshot of the present screen.

### CODE REVIEW

File: Java/ScreenshotTaker/screenshottaker.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    public static final long serialVersionUID = 1L; 
+    public static void main(String[] args) 
+    { 
+        try 
+        {
+            Thread.sleep(120); //sleeping the main thread for 120 ms
+            Robot r = new Robot(); 
+
+            //to save the screen shot to a specific location
+            String path = "D:// Screenshot.jpg";
```
Comment (tripathyprateek) [inline Java/ScreenshotTaker/screenshottaker.java:21]: As far as I can understand, this line will generate all screenshot with screenshot.jpg as the name. Can we make the naming of screenshot variable? Just try adding the time screenshot was taken to the end of the file to make it variable if possible.

Comment (iamvs-2002) [reply Java/ScreenshotTaker/screenshottaker.java:21]: Yea sure...rather than that we can ask the user to decide the name of the screenshot.

**Comment by iamvs-2002** [comment]
@tripathyprateek Done! I have asked the user to enter the name of the file for the screenshot.

**Comment by tripathyprateek** [comment]
cool @iamvs-2002 . sorry for the late merge, this got lost in the notifications. Merging now, and make sure to update the Google Sheet.

**Comment by iamvs-2002** [comment]
> cool @iamvs-2002 . sorry for the late merge, this got lost in the notifications. Merging now, and make sure to update the Google Sheet.

Thanks a lot buddy. And yea, I have updated the same in the spreadsheet!

**Pull request merged**